<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependencyTreeWidget</class>
 <widget class="QWidget" name="DependencyTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Entries</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="DactListView" name="fileListWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
            <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
            </property>
            <addaction name="copyFileNamesAction" />
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="filterProgressBar">
            <property name="visible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="statisticsGroupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Statistics</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="statisticsLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::FieldsStayAtSizeHint</enum>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="hitsDescLabel">
              <property name="text">
               <string>Hits:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="hitsLabel">
              <property name="toolTip">
               <string>Number of matching nodes found in the sentences</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="entriesLabel">
              <property name="toolTip">
               <string>Number of sentences</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Entries:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Tree</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QLineEdit" name="highlightLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Highlight:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DactTreeView" name="treeGraphicsView" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Sentence:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BracketedSentenceWidget" name="sentenceWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>200</horstretch>
             <verstretch>24</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="copyFileNamesAction">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <enum>QKeySequence::Copy</enum>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DactListView</class>
   <extends>QWidget</extends>
   <header>DactListView.hh</header>
  </customwidget>
  <customwidget>
   <class>DactTreeView</class>
   <extends>QWidget</extends>
   <header>DactTreeView.hh</header>
  </customwidget>
  <customwidget>
   <class>BracketedSentenceWidget</class>
   <extends>QFrame</extends>
   <header>BracketedSentenceWidget.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>copyFileNamesAction</sender>
   <signal>triggered()</signal>
   <receiver>DependencyTreeWidget</receiver>
   <slot>copy()</slot>
  </connection>
  <connection>
   <sender>fileListWidget</sender>
   <signal>customContextMenuRequested(const QPoint &amp;)</signal>
   <receiver>DependencyTreeWidget</receiver>
   <slot>showToolMenu(const QPoint &amp;)</slot>
  </connection>
 </connections>
</ui>
